src/value/mod.rs

```diff
@@ -177,22 +180,24 @@ impl Value {
     }
 }
 
-impl FromValue for ValueMap {
-    fn from_value(value: &mut ValueMap) -> Option<&mut Self> {
-        Some(value)
-    }
-}
+// impl<'b> FromValue for ValueMap<'b> {
+//     fn from_value<'a>(value: &'a mut ValueMap) -> Option<&'a mut Self>
+//     where 'b:'a
+//     {
+//         Some(value)
+//     }
+// }
```

Comment:
Important steps should not lose

Reply:
impl `FromValue` for `ValueMap` can't be done due to trait `Rult`. Close this PR.